example/e2e/init.js

```diff
@@ -0,0 +1,25 @@
+const detox = require('detox');
+const config = require('../../package.json').detox;
+const adapter = require('detox/runners/jest/adapter');
+const specReporter = require('detox/runners/jest/specReporter');
+
+// Set the default timeout
+jest.setTimeout(120000);
+jasmine.getEnv().addReporter(adapter);
```

Comment:
where is `jasmine` coming from here? Does detox provide that on `global` by default?

Reply:
`jest` is actually using `jasmine` so it's coming from `jest/adapter`

Reply:
gotta luv test globals 😁